Output a helpful error message if the SSL module could not be loaded.

Index: ddclient-3.7.0/ddclient
===================================================================
--- ddclient-3.7.0.orig/ddclient	2007-01-20 11:42:10.448629056 +0100
+++ ddclient-3.7.0/ddclient	2007-01-20 13:48:43.142316029 +0100
@@ -1638,6 +1638,20 @@
     $res;
 }
 ######################################################################
+## load_ssl_support
+######################################################################
+sub load_ssl_support {
+    my $ssl_loaded = eval {require IO::Socket::SSL};
+    unless ($ssl_loaded) {
+        fatal(<<"EOM");
+Error loading the Perl module IO::Socket::SSL needed for SSL connect.
+On Debian, the package libio-socket-ssl-perl must be installed.
+EOM
+    }
+    import  IO::Socket::SSL;
+    { no warnings; $IO::Socket::SSL::DEBUG = 0; }
+}
+######################################################################
 ## geturl
 ######################################################################
 sub geturl {
@@ -1666,9 +1680,7 @@
     if ( $globals{'ssl'} and (caller(1))[3] ne 'main::get_ip' ) {
         $use_ssl      = 1;
         $default_port = 443;
-        require IO::Socket::SSL;
-        import  IO::Socket::SSL;
-        { no warnings; $IO::Socket::SSL::DEBUG = 0; }
+        load_ssl_support;
     } else {
         $use_ssl      = 0;
         $default_port = 80;
